<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-858PIU1" failures="2" tests="2" name="Default test(failed)" time="0.217" errors="0" timestamp="20 Mar 2022 08:59:30 GMT">
  <testcase classname="A4_FailedscreenShot.A18_RunOnlyFailedTestCase" name="mul" time="0.08">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zeroat A4_FailedscreenShot.A18_RunOnlyFailedTestCase.mul(A18_RunOnlyFailedTestCase.java:22)... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- mul -->
  <testcase classname="A4_FailedscreenShot.A18_RunOnlyFailedTestCase" name="sub" time="0.009">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zeroat A4_FailedscreenShot.A18_RunOnlyFailedTestCase.sub(A18_RunOnlyFailedTestCase.java:17)... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- sub -->
  <testcase classname="A4_FailedscreenShot.A18_RunOnlyFailedTestCase" name="add" time="0.0">
    <ignored/>
  </testcase> <!-- add -->
  <testcase classname="A4_FailedscreenShot.A18_RunOnlyFailedTestCase" name="div" time="0.0">
    <ignored/>
  </testcase> <!-- div -->
</testsuite> <!-- Default test(failed) -->
